app/_locales/zh_TW/messages.json

```diff
@@ -924,8 +1117,36 @@
   "stateLogsDescription": {
     "message": "狀態紀錄包含您的公開帳戶位址和已傳送的交易資訊"
   },
```

Comment:
```suggestion
  },
  "statusConnected": {
    "message": "已連接"
  },
  "statusNotConnected": {
    "message": "未連接"
  },
```

Reply:
Maybe help me compare `zh_TW` with `zh_CN` to check is there any other translation missing